examples/webgpu_skinning.html

```diff
@@ -54,38 +54,27 @@
 				camera.position.set( 1, 2, 3 );
 
 				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 'lightblue' );
 				camera.lookAt( 0, 1, 0 );
 
 				clock = new THREE.Clock();
 
 				//lights
 
 				const light = new THREE.PointLight( 0xffffff, 1, 100 );
-				light.power = 1700; // 100W
+				light.power = 2500;
 				camera.add( light );
 				scene.add( camera );
 
-				const loader = new FBXLoader();
-				loader.load( 'models/fbx/Samba Dancing.fbx', function ( object ) {
-
-					object.scale.setScalar( .01 );
+				const loader = new GLTFLoader();
+				loader.load( 'models/gltf/Michelle.glb', function ( gltf ) {
 
+					const object = gltf.scene;
 					mixer = new THREE.AnimationMixer( object );
 
-					const action = mixer.clipAction( object.animations[ 0 ] );
+					const action = mixer.clipAction( gltf.animations[ 0 ] );
 					action.play();
 
-					object.traverse( function ( child ) {
```

Comment:
Why did you remove this?

Reply:
so the character would show its materials.

With those lines

<img width="1038" alt="Screenshot 2023-03-11 at 08 30 53" src="https://user-images.githubusercontent.com/234804/224496112-0deb5cb9-36e8-4d0e-9f2d-e239ff590b34.png">

Without those lines

<img width="1038" alt="Screenshot 2023-03-11 at 08 29 32" src="https://user-images.githubusercontent.com/234804/224496120-19a70c57-f274-4bc6-958b-e601eeb266c5.png">

Reply:
OK. Please remove `MeshStandardNodeMaterial` import then.

Reply:
done